### ISSUE TITLE
Idea: Include Link Checker module for automatic link validation

### ISSUE THREAD

**Issue by jonstahl** [givingforum/givingforum#197]
Given that our sites are now becoming very content rich, it might make sense to fend off broken/obsolete links with https://www.drupal.org/project/linkchecker.

**Comment by kristenruff** [comment]
Seems like a good suggestion.

**Comment by PhilOhio** [comment]
Sounds good- POH

**Comment by mtrimbach** [comment]
Agree that it seems like a good suggestion.

**Comment by vrozansky** [comment]
Checked with ThinkShout on this and here's their feedback (from Nancy):

"I have not used that module before, but I spoke to some of our team about it. We do have some experience with it, but would not recommend adding it to the build because it is a rather complex module that has to run with cron and process the entire database. There are plenty of third party services out there that can be run externally occasionally on the site that might be a better option. For starters, you might try https://validator.w3.org/checklink. That one is free. There are also paid services out there."

**Comment by jonstahl** [comment]
Fair enough. I may experiment with this independently. I’m not afraid of running things with cron once in a while. ☺

**Comment by sgeis** [comment]
Indiana is interested in learning more.
